packages/nextra/src/server/remark-plugins/remark-structurize.ts

```diff
@@ -56,10 +59,18 @@ export const remarkStructurize: Plugin<[Search], Root> = options => {
       }
     } else if (
       (opts.codeblocks && type === 'code') ||
-      ['text', 'inlineCode', 'tableCell'].includes(type)
+      type === 'text' ||
+      type === 'inlineCode' ||
+      type === 'tableCell'
     ) {
-      result += node.value
-      if (!skip) content += node.value
+      // Because of markdown extensions, inline code may have a `{:type}` suffix.
+      // Trim this off if it exists.
+      const value =
+        type === 'inlineCode'
+          ? node.value.replace(/\s*\{:\w+\}$/, '')
```

Comment:
```suggestion
      // Inline code may have a `{:language}` suffix, trim this off if it exists.
      const value =
        type === 'inlineCode'
          ? node.value.replace(/\{:\w+\}$/, '')
```

i think `\s*` is useless here